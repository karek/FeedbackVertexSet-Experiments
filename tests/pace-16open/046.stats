73,152,8
