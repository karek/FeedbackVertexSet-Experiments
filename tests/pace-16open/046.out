18
52 51 40 50 29 57 69 31 35 38 6 18 59 28 11 21 12 25 
